enterprise/embed-media/index.md

```diff
@@ -14,16 +14,16 @@ It is now as simple as adding a plugin to deliver a modern, content creation exp
 
 Your have two deployment options. One is super easy with {{site.cloudname}}, the other is to go the more traditional Self-hosted route, requiring an additional [server-side component]({{ site.baseurl }}/enterprise/server/) to be installed and configured.
 
-## Enhanced Media Embed cloud setup
-
-[{{site.cloudname}}]({{ site.baseurl }}/cloud-deployment-guide/editor-and-features/) makes setting up Enhanced Media Embed a breeze. Simply include the `mediaembed` parameter in your `tinymce.init` and have the {{site.cloudname}} services do the work for you.
-
 ## Enhanced Media Embed self-hosted setup
 
-If you'd rather deploy Enhanced Media Embed via the Self-hosted package, you have a little more work to do. Once you've got the [server-side component]({{ site.baseurl }}/enterprise/server/) installed, additional configuration to your `application.conf` file is required. (Don't forget to restart the Java application server after updating the configuration.)
+Once you've got the [server-side component]({{ site.baseurl }}/enterprise/server/) installed, additional configuration to your `application.conf` file is required. (Don't forget to restart the Java application server after updating the configuration.)
```

Comment:
I would have thought we'd prefer "you have" rather than "you've got" because the latter is colloquial?

Reply:
Updated all instances of "Once you've" to "Once you have".